tests/rustdoc/demo-allocator-54478.rs

```diff
@@ -40,6 +40,7 @@
 //! }
 //!
 //! fn main() {
+//!     drop(std::env::var("PWD"));
```

Comment:
```suggestion
//!     drop(String::from("an allocation"));
```
This is not only clearer but will also work when `$PWD` doesn't exist.